### ISSUE TITLE
[FEATURE] Review if suggested modules work correctly with Lancer module

### ISSUE THREAD

**Issue by TheFwuffernaut** [Eranziel/foundryvtt-lancer#202]
**Is your feature request related to a problem? Please describe.**
Token Tool-tip Alt is not working. Additional modules that are based on the core Lancer module (referencing values added, TTA uses "mech.") may also have this issue (untested yet).

**Describe the solution you'd like**
For the Lancer module to use the values referenced in the default setup for module/s.
ALTERNATIVELY
Quick primer on how to set up alternate module/s if they do in fact work but the defaults are not intended.

**Describe alternatives you've considered**
See above.

**Additional Details**
Tests performed:
- importing from CC/creating a pilot manually (netiher work. Tested this because of issues with being unable to edit things or things unlinking when tokens were moved)
- using alternative things that display values (brawlbars)
- changing value names in module
- changing module value find location

None of the above worked.

**Comment by Eranziel** [comment]
Is this for beta or stable release? Both versions can already handle the issues mentioned here, but the solution is slightly different for each.

**Comment by TheFwuffernaut** [comment]
Beta branch 0.9.x (for post Christmas CC LCPs). 

Initial setup didn't work, nor did attempting to swap to derived.whatever

Assuming I did something wrong with the latter however as someone else took a look and got it working with the same formatting (perhaps a settings thing that I missed or some code I didn't know needed to be inserted?). 

The bogstandard it comes with (mech.) definitely doesn't work however and took some fiddling to make work, hence the suggestion to review any defaults that its sending out to modules in a similar manner.

**Comment by ScyllPoesis** [comment]
For TTA, that's in their code, not ours, and we will not update it until we have a stable release. That's just how that works. In general while in beta any external repositories will need to be forked with beta support branches (Like for TAH: https://raw.githubusercontent.com/ScyllPoesis/fvtt-tokenactionhud/lancer-1.0/module.json) or manually configured (like TTA)

**Comment by TheFwuffernaut** [comment]
Is there at least the possibility of some documentation somewhere so that we can actually test these things and you know what you'll be implementing come the stable release?

I understand not rolling an update out immediately but also if people don't know how to set things up they can't test things for you.

**Comment by ScyllPoesis** [comment]
Yeah, we have pretty poor documentation at this point. However I'd answer that:
1. This version has jumped not one but two backing changes (lancer-0.1 -> lancer-0.9 and now fvtt0.7 -> fvtt0.8). This means we've changed the various backing structures and almost all of our time has been on building this stuff out. We're always happy for testing but honestly the past week or two of reports and issues will need to be retested due to the scope of all our changes in the pursuit of the FVTT version migration, and the fact that the version you're testing is pretty behind our dev version (unless you've been building from scratch, in which case carry on).
2. We're lazy and with a lack of hands over the past couple weeks I basically just had the time to answer specific questions on discord and not really write up a doc on the wiki. External code is also not the main focus of our development here, and so when testing out the system itself whether or not TAH or TTA is working correctly is not our priority, we're basically just trying to bolt everything here together first. If we have time I'm sure someone is happy to make some tweaks (like I did with the fork above), but there should not be an expectation that our beta versions will always be compatible or functional with external modules.

tl;dr our documentation is bad 'cause we're volunteers and spending all our time slapping this beast around at the moment, the beta is focused on testing the foundry system, external modules not guaranteed to work.

**Comment by TheFwuffernaut** [comment]
Fair enough, may want to slap a "These have a 90%+ chance of not working on the new version, please don't bother us about them yet." on the recommended modules list that's been floating around then or something idk.

**Comment by ScyllPoesis** [comment]
Yeah, there's a ton I would've personally done a lot differently about this sprint, but life gets hectic and we're hopeful that the end result will make up for the aggravation you guys may have felt during this testing period. It's really just been a crazy time thanks to us pivoting to jump to fvtt0.8 as we were not expecting it to come out this month. Once we get our next beta release out this week or the next (hopefully final pre-RC), I'll try to lay out a nice doc with what I'm using and explanation of the new paths and structures.
